<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CBootScreenFrame</class>
 <widget class="QWidget" name="CBootScreenFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>647</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>647</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>647</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>CBootScreenWidget</string>
  </property>
  <widget class="QLabel" name="label_first">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>300</y>
     <width>291</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 9pt &quot;微软雅黑&quot;;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_second">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>326</y>
     <width>291</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 8pt &quot;宋体&quot;;
color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_third">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>349</y>
     <width>291</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 8pt &quot;宋体&quot;;
color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QWidget" name="main_widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>647</width>
     <height>400</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>647</width>
     <height>400</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>647</width>
     <height>400</height>
    </size>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>55</x>
      <y>50</y>
      <width>251</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>9</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 9pt &quot;宋体&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>FreeViewer是基于FreeEarth三维数字地球开发</string>
    </property>
    <property name="indent">
     <number>-1</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>261</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 9pt &quot;宋体&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>平台打造的一款功能强大的桌面应用。</string>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>39</x>
      <y>110</y>
      <width>171</width>
      <height>171</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_roundBar">
     <property name="spacing">
      <number>0</number>
     </property>
    </layout>
   </widget>
   <widget class="QWidget" name="flash_widget" native="true">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>70</y>
      <width>279</width>
      <height>272</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>160</y>
      <width>141</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_ballPath"/>
   </widget>
  </widget>
  <zorder>main_widget</zorder>
  <zorder>label_first</zorder>
  <zorder>label_second</zorder>
  <zorder>label_third</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../resource/images.qrc"/>
 </resources>
 <connections/>
</ui>
